vassal-doc/src/main/readme-referencemanual/ReferenceManual/NamedKeyCommand.adoc

```diff
@@ -1,31 +1,48 @@
 If the trait is also assigned a Menu Command string, then the keystroke for the Key Command will be displayed in the context menu for the piece as a shortcut.
 
-*Named Commands:* A Named Command, by contrast, can be any text message at least two characters long, such as FireForwardPhasers or EliminateUnit or SpinTurretRight.
+==== Named Commands
+A Named Command, by contrast, can be any text message at least two characters long, such as FireForwardPhasers or EliminateUnit or SpinTurretRight.
 These names will not be displayed on screen during play, they are private to the module, and function much like naming a variable, method, or function in a programming language.
 If a Named Command is paired with a Menu Command string, then it will still generate an entry on the piece's context menu, but no keystroke will be displayed.
 If the player then selects that menu item, the Named Key Command will "fire" and any trait in the piece with that Named Key Command assigned will be activated.
 
-*Activating Key Commands:* In addition to being activated when the player selects the corresponding menu item from the context menu, and (in the case of regular Key Commands) when the player presses the designated keystroke combination while the piece is selected, both types of Key Command can _also_ be generated by a <<TriggerAction.adoc#top,Trigger Action>> trait from the same piece, or by a <<GlobalKeyCommand.adoc#top,Global Key Command>> from any piece.
+==== Activating Key Commands
+In addition to being activated when the player selects the corresponding menu item from the context menu, and (in the case of regular Key Commands) when the player presses the designated keystroke combination while the piece is selected, both types of Key Command can _also_ be generated by a <<TriggerAction.adoc#top,Trigger Action>> trait from the same piece, or by a <<GlobalKeyCommand.adoc#top,Global Key Command>> from any piece.
 This allows module designers to chain several actions together to create sophisticated modules.
 
 Earlier versions of Vassal supported _only_ traditional Key Commands (i.e.
 actual keystrokes), and so every aspect of a trait of any piece potentially needing to be activated had to be assigned its own keystroke.
 This quickly made the keyboard seem very crowded for designers of sophisticated modules! So the Named Command was introduced to allow modules more breathing room to create useful traits and commands.
-|image:images/CounterGlobalKeyCommand.png[]  +
-|===
+
+==== Editing Key Commands
+image::images/KeyCommandEntry.png[]
+
+The standard Key Command configurer contains 3 parts::
++
+*1:* Named Command entry. Type or paste the text of a Name Command in this section.
++
+*2:* Keystroke Command entry. Click in this section, then press a Keyboard key or key combination. The name of the Keystroke Command will be displayed.
```

Comment:
Should we mention that typing in the Named Command field will clear the Keystroke Command field, and vice versa?  I'm not sure if it's already clear that you can't have both....

Reply:
Good idea. Note added.